### PR TITLE
Add missed dsl html for ProcessResources task

### PR DESCRIPTION
**Pull request by 6hundreds**
Fixes https://github.com/gradle/gradle/issues/21522

### CODE REVIEW

**Review by nathan-contino — APPROVED** [review]
LGTM assuming this has the intended result. You can run `@bot-gradle test BD` to be sure before you merge.

**Review by ljacomet — CHANGES_REQUESTED** [review]
Could you make this PR against `release` and target 7.6 RC1?

Changes are ✅ otherwise

**Review by ljacomet — APPROVED** [review]

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Pull request merged**
